cmd/collectors.go

```diff
@@ -27,16 +27,20 @@ import (
 	"github.com/kelseyhightower/envconfig"
 	"github.com/loadimpact/k6/lib"
 	"github.com/loadimpact/k6/stats/cloud"
+	"github.com/loadimpact/k6/stats/dogstatsd"
 	"github.com/loadimpact/k6/stats/influxdb"
 	jsonc "github.com/loadimpact/k6/stats/json"
+	"github.com/loadimpact/k6/stats/statsd"
 	"github.com/pkg/errors"
 	"github.com/spf13/afero"
 )
 
 const (
-	collectorInfluxDB = "influxdb"
-	collectorJSON     = "json"
-	collectorCloud    = "cloud"
+	collectorInfluxDB  = "influxdb"
+	collectorJSON      = "json"
+	collectorCloud     = "cloud"
+	collectorStatsD    = "statsd"
+	collectorDogStatsD = "dogstatsd"
```

Comment:
Hm, I kinda feel like "datadog" would be a more obvious name here.

Reply:
I think that the only benefit of having as datadog is that it's a shorter name, other than that statsd and dogstatsd makes more sense in terms of convention because in practice it's an implementation of a statsd protocol. Both implement statsd protocal but dogstatsd extends it. I believe is more transparent that way.

Reply:
What I'm thinking is that most people will come looking for "datadog" and instead find "dogstatsd", then get confused by what the difference is, while people who are running dogstatsd know what they're doing. I could be wrong.

Reply:
That's a fair point. Do you think that can be solved with documentation?

Reply:
Possibly, but I feel like it's better to avoid the confusion in the first place.

Unfortunately, a lot of people seem to *really* hate reading the effing manual.

Reply:
@liclac  I've updated the collector name to be datadog